components/brave_rewards/resources/android_page/components/pageWallet.tsx

```diff
@@ -56,7 +56,8 @@ class PageWallet extends React.Component<Props, State> {
     }
 
     let claimedPromotions = promotions.filter((promotion: Rewards.Promotion) => {
-      return promotion.status === 4 // PromotionStatus::FINISHED
+      return promotion.status === 4 || // PromotionStatus::FINISHED
+        promotion.status === 5 // PromotionStatus::OVER
```

Comment:
nit: @NejcZdovc not for this PR, but `::OVER` would be better named `EXPIRED`

Reply:
@NejcZdovc Business logic like this should be moved to the libs (but not for this ticket as unrelated)

Reply:
it's not a business logic. We send promotions to the UI and then UI decides what to show where. So for example from the same list we show notifications about new grants and list of already claimed grants

Reply:
cc @kylehickinson for visibility